src/components/view/MedicationsView/MedicationsView.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Layout, Card, NavigationBar, MedicationsList, Title } from "../.."
+import { TermInformation } from '../../presentational/LabResultWithSparkline/LabResultWithSparkline';
+import MyDataHelps from '@careevolution/mydatahelps-js';
+import medicationIcon from "../../../assets/icon-medication.svg";
+import language from '../../../helpers/language';
+
+export interface MedicationsViewProps {
+    presentation?: "Push" | "Modal"
+    previewState?: "default"
+}
+
+export default function (props: MedicationsViewProps) {
+    function viewTermInfo(termInfo: TermInformation) {
+        var queryString = new URLSearchParams({ termFamily: termInfo.TermFamily, termNamespace: termInfo.TermNamespace, termCode: termInfo.TermCode, lang: MyDataHelps.getCurrentLanguage() }).toString();
+        MyDataHelps.openApplication("https://hw.careevolutionapps.com/TermInformation.html?" + queryString, { modal: true });
```

Comment:
Is there a way to try this out for me?

Reply:
Not super easily - but this is the same code invoked by the main H&W dashboard for selecting term information from a lab result etc.